receipt_dynamo/receipt_dynamo/data/_receipt_word_label_spatial_analysis.py

```
@@ -222,7 +213,7 @@ def get_receipt_word_label_spatial_analysis(
             )
         if not isinstance(receipt_id, int):
             raise EntityValidationError(
-                f"receipt_id must be an integer, got {type(receipt_id).__name__}"
+                f"receipt_id must be an int, got {type(receipt_id).__name__}"
             )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)